include/zephyr/sensing/sensing.h

```diff
@@ -38,16 +38,20 @@ extern "C" {
  */
 struct sensing_sensor_version {
 	union {
-		uint32_t value;
+		uint32_t value; /** The version represented as a 32-bit value. */
```

Comment:
Not sure why all the `/**< ...  **/` style comments are gone: you need them when you want to indicate the comment applies to the compilation unit that's *before* (hence `<`) the comment

Reply:
Updated.